src/btree/bt_debug.c

```diff
@@ -135,7 +137,7 @@ __debug_item_key(WT_DBG *ds, const char *tag, const void *data_arg, size_t size)
     session = ds->session;
 
     return (ds->f(ds, "\t%s%s{%s}\n", tag == NULL ? "" : tag, tag == NULL ? "" : " ",
-      F_ISSET(ds, WT_DEBUG_UNREDACT) ?
+      (F_ISSET(ds, WT_DEBUG_UNREDACT_ALL) || F_ISSET(ds, WT_DEBUG_UNREDACT_KEYS)) ?
```

Comment:
even this should work -  `F_ISSET(ds, WT_DEBUG_UNREDACT | WT_DEBUG_UNREDACT_KEYS)`